app/helpers/banner_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "yaml"
+
+module BannerHelper
+  # rubocop:disable Rails/ContentTag
+  def banner_content
+    @yaml_data = YAML.load_file("config/banner.yml")
+    return false if @yaml_data[Rails.env].nil?
+    @banner_title = "<h1>#{@yaml_data[Rails.env]['title']}</h1>"
+    @banner_body = "<p>#{@yaml_data[Rails.env]['body']}</p>"
```

Comment:
I would love it if the `<p>` went into the partial
```suggestion
    @banner_body = @yaml_data[Rails.env]['body']
```